packages/p2p/src/components/orders/orders.scss

```diff
@@ -52,6 +67,68 @@
             max-width: 100%;
         }
     }
+    &__expander {
+        padding: 1.6rem;
+        border-top: 1px solid var(--general-section-1);
+        &-heighlight-danger {
+            background-color: #fff8f8;
+        }
+        & .dc-table__row {
+            border-bottom: unset;
+        }
+        &-separator {
+            width: 100vw;
+            height: 1px;
+            border-top: 1px solid var(--general-section-1);
+            position: absolute;
+            left: 0rem;
+            margin: 1.6rem 0rem;
+        }
+        &-header {
+            display: flex;
+            justify-content: space-between;
+            margin: 0.4rem 0rem 0.8rem;
+            align-items: center;
+
+            &-right {
+                display: flex;
+            }
+        }
+        &-content {
+            display: grid;
+            margin-top: 1.6rem;
+            grid-template-columns: repeat(2, 1fr);
+            align-items: center;
+            grid-auto-rows: 6.2rem;
+            padding: 0.8rem;
+            &__title {
+                margin-bottom: 1.6rem;
+            }
+        }
+
+        &-chat {
+            margin: 0rem 0.9rem;
+        }
+        &-time {
+            border: solid 1px var(--border-normal);
+            border-radius: 1.6rem;
+            width: 7.8rem;
+            height: 2.4rem;
+            padding: 0.2rem;
+            text-align: center;
+            font-size: var(--text-size-xxs);
+            letter-spacing: 1px;
+        }
+        &-status {
+            width: auto;
+            padding: 0rem 1.6rem;
+            height: 2.4rem;
+            font-weight: bold;
+            text-align: center;
+            font-size: var(--text-size-xs);
```

Comment:
Can we wrap the content of this in a `Text` component?

Reply:
because we defined different status background-color and color for the status by wrapping it to text. we still need to define a class to update styles